go/worker/merge/committee/node.go

```diff
@@ -358,7 +358,14 @@ func (n *Node) tryFinalizeResultsLocked(pool *commitment.Pool, didTimeout bool)
 	// NOTE: The roothash backend will start counting its timeout on its own based on
 	//       any received commits so in the worst case the actual timeout will be
 	//       2*roundTimeout.
-	roundTimeout := n.commonNode.Roothash.Info().ComputeRoundTimeout
+	params, err := n.commonNode.Roothash.ConsensusParameters(ctx, 0)
```

Comment:
We should probably query at epoch height (e.g., `epoch.GetGroupVersion()` where `epoch` can be passed by the caller).

Reply:
moved it into Group `epoch` & `epochsnapshot`, and is now queried on epoch transition.